inc/css/class-css-handler.php

```diff
@@ -443,6 +446,28 @@ public static function compress( $css ) {
 		return $css;
 	}
 
+	/**
+	 * Mark in post meta if the post has a review block.
+	 * 
+	 * @param int $post_id Post ID.
+	 * @since 2.4.0
+	 * @access public
+	 */
+	public static function mark_review_block_metadata( $post_id ) {
+		if ( empty( $post_id ) ) {
+			return;
+		}
+
+		$content = get_the_content( '', false, $post_id );
+
+		if ( empty( $content ) ) {
+			return;
+		}
+
+		$has_review = false !== strpos( $content, '<!-- wp:themeisle-blocks/review' );
```

Comment:
@HardeepAsrani, is there a problem if we use this fast checking instead of parsing the blocks?

Reply:
No, looks good. You can just test to make sure it works in all contexts.